swap_meet/decor.py

```diff
@@ -1,2 +1,11 @@
-class Decor:
-    pass
\ No newline at end of file
+
+from swap_meet.item import Item
+class Decor(Item):
+    def __init__(self, width = 0, length = 0, id = None, condition = 0, age = 0):
+        super().__init__(condition, id, age)
```

Comment:
Nice work with the `super` function, before using this we know that to get the same behavior we would have to write a bunch of the same code, which doesn't fall in line with our `DRY` paradigm, this saves us a lot of typing and causes the code base to much more concise. Also, great work on passing in the `None` default param in the `__init__` function.